### DISCUSSION TITLE
MRTK 2.8.3 patch release date?

### DISCUSSION THREAD

**Discussion by ptc-agrill** [microsoft/MixedRealityToolkit-Unity#11083] [General]
hi,

I would like to know if there's a MRTK 2.8.3 hotfix release planned in the near future?

There's a bug with MRTK 2.8.2 that makes button clicks unavailable on mobile platforms that has been fixed and mainlined about 2 weeks ago with this PR: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11036

We'd appreciate to be able to use an official MRTK release and not fork the project just for this one fix.

Best regards,
Alex

**Comment by MaxWang-MS** [comment]
Sorry for the late response, but we are looking at a release in a week or two.

**Reply by ptc-agrill** [reply]
thanks for the update!
